heudiconv/dicoms.py

```diff
@@ -62,6 +62,14 @@ def create_seqinfo(mw, series_files, series_id):
     global total_files
     total_files += len(series_files)
 
+    custom_seqinfo_data = custom_seqinfo(series_files=series_files, wrapper=mw) \
+        if custom_seqinfo else None
+    try:
+        hash(custom_seqinfo_data)
+    except TypeError:
+        raise RuntimeError("Data returned by the heuristics custom_seqinfo is not hashable. "
+        "See https://heudiconv.readthedocs.io/en/latest/heuristics.html#custom_seqinfo for more details.")
```

Comment:
@bpinsard so WDYT about adding some check for roundtrip serialization into/from .tsv? a "weaker" alternative could be to test for presence of `\t` or `\n` symbols which might break it, or they wouldn't ?  

apparently we did not add any test yet here, should have that.

Reply:
Is there any part of heudiconv that might load the `dicominfo*.tsv` that get written in `.heudiconv`, in a second run with overwrite for instance?

For now it convert `custom` values to a str, eg. when using frozendict for hashable dict:
`frozendict.frozendict({'patient_name': 'p02_ana012', 'pe_dir': 'ROW', 'pe_dir_pos': 1, 'body_part': 'BRAIN', 'scan_options': "['IR', 'WE']", 'image_comments': '', 'slice_orient': 'Sag', 'echo_number': 'None', 'rescale_slope': None, 'receive_coil': 'HC1-7;NC1,2', 'image_history': 'ChannelMixing:ND=true_CMM=1_CDM=1;ACCAlgo:9', 'ice_dims': 'X_1_1_1_1_1_4_1_1_1_1_1_35'})` 

So not necessarily easily unserializable depending on what the custom_seqinfo is returning.